tools/bridge/docker/docker-compose-base.yaml

```diff
@@ -1,7 +1,7 @@
 version: "2.4"
 services:
   node_foreign:
-    image: parity/parity:v2.5.5-stable
+    image: parity/parity:stable
```

Comment:
Do you know what this file is used for?

Reply:
I'm not entirely sure, but I don't think we use it for anything. Validators could use it though if they don't want to use the quickstart script. Don't think it's strictly necessary though.